docs/memcached.md

```diff
@@ -1,39 +1,11 @@
-# `memcached` Metrics Receiver
-
-The memcached receiver can retrieve stats from your memcached server through the . 
-
-
-## Configuration
-
-Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your memcached configuration.
-
-To configure a receiver for your memcached metrics, specify the following fields:
-
-| Field                 | Default                   | Description |
-| ---                   | ---                       | ---         |
-| `type`                | required                  | Must be `memcached`. |
-| `endpoint`            | `localhost:3306`          | The url, or unix socket file path, for your memcached server. |
-| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
-
-Example Configuration:
-
-```yaml
-metrics:
-  receivers:
-    memcached_metrics:
-      type: memcached
-      collection_interval: 60s
-  service:
-    pipelines:
-      memcached_pipeline:
-        receivers:
-          - memcached_metrics
-```
+# Memcache
 
+Follow https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/memcached
+for instructions to collect logs and metrics from this application using Ops Agent.
```

Comment:
Follow the [installation guide](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/memcached) for instructions to collect logs and metrics from this application using Ops Agent.

Reply:
Sure thing. Changed.